src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c

```diff
@@ -110,8 +104,8 @@ s32 EnBomChu_UpdateFloorPoly(EnBomChu* this, CollisionPoly* floorPoly, GlobalCon
 
     this->actor.floorPoly = floorPoly;
 
-    // This NULL check means if the player releases a Bombchu
-    // out of bounds, the game will not crash, unlike OoT.
+    // This NULL check means if the player releases a Bombchu out of bounds,
```

Comment:
I don't think "out of bounds" is well-defined... you mean something like "with no collision below" or better wording.

Reply:
Yeah I was sort of writing that from a speedrunner's perspective, which is imprecise. Let me fix that.